src/NHapi.Base/Parser/ParserBase.cs

```diff
@@ -393,23 +384,18 @@ public static String GetMessageStructureForEvent(String name, String version)
 		}
 
 
-		/// <summary> Note that the validation context of the resulting message is set to this parser's validation 
-		/// context.  The validation context is used within Primitive.setValue().
-		/// 
+		/// <summary>
+		/// Note that the validation context of the resulting message is set to this parser's validation 
+		/// context. The validation context is used within Primitive.setValue().
 		/// </summary>
-		/// <param name="name">name of the desired structure in the form XXX_YYY
-		/// </param>
-		/// <param name="version">HL7 version (e.g. "2.3")  
-		/// </param>
-		/// <param name="isExplicit">true if the structure was specified explicitly in MSH-9-3, false if it 
+		/// <param name="theName">name of the desired structure in the form XXX_YYY.</param>
+		/// <param name="theVersion">HL7 version (e.g. "2.3").</param>
+		/// <param name="isExplicit">
+		/// true if the structure was specified explicitly in MSH-9-3, false if it 
 		/// was inferred from MSH-9-1 and MSH-9-2.  If false, a lookup may be performed to find 
-		/// an alternate structure corresponding to that message type and event.   
+		/// an alternate structure corresponding to that message type and event.
 		/// </param>
-		/// <returns> a Message instance 
-		/// </returns>
-		/// <throws>  HL7Exception if the version is not recognized or no appropriate class can be found or the Message  </throws>

```

Comment:
I think you remove the excetion block by accident although the syntax is wrong here.